fabric/bin/lib/common_ledger.sh

```diff
@@ -108,9 +108,20 @@ ledger_init() {
     pushd ${FABRIC_CFG_PATH}
 
     # 1. clean up any prior state
+    #     - ledger state and log files from peer & orderer
     [ ! -z "${FABRIC_STATE_DIR}" ] || die "FABRIC_STATE_DIR not defined" # just in case ..
     try mkdir -p ${FABRIC_STATE_DIR}
     try rm -rf ${FABRIC_STATE_DIR}/*
+    #     - external builder artifacts
+    #       Note: FABRIC_STATE_DIR which will be ledger specific and allows for
+    #         multiple concurrent ledgers (assuming their core.yaml have 
+    #         different fileSsystemPaths) these artifacts are also shared 
+    #         _across_ ledger definitions (core.yaml).
+    #         Contrary to that, external builder are ledger instance agnostic, hence we also
+    #         provide an opt-out for the wiping to allow running current ledgers ..  
+    if [ -z ${FPC_SKIP_EXTBUILDER_CLEANUP+x} ]; then
```

Comment:
so with this setting, it cleans up the tmp folder at every execution of the integration test. right?

Reply:
unless you set the corresponding `FPC_SKIP_EXTBUILDER_CLEANUP` env variable, it will clean-up

Reply:
sounds good